simulation/traffic_simulator/include/traffic_simulator/math/catmull_rom_spline.hpp

```diff
@@ -52,7 +52,8 @@ class CatmullRomSpline
     geometry_msgs::msg::Point point0, geometry_msgs::msg::Point point1,
     bool search_backward = false) const;
   boost::optional<double> getCollisionPointIn2D(
-    std::vector<geometry_msgs::msg::Point> polygon, bool search_backward = false) const;
+    std::vector<geometry_msgs::msg::Point> polygon, bool search_backward = false,
+    bool close_start_end = true) const;
```

Comment:
std::vector<geometry_msgs::msg::Point> => const std::vector<geometry_msgs::msg::Point> &

Reply:
fixed in eba2616447eac0694651a2ab4a1298df7d8a44d8